parcels/field.py

```diff
@@ -184,6 +196,7 @@ def from_netcdf(cls, filenames, variable, dimensions, indices=None, grid=None,
                fully load them (default: True). It is advised to deferred load the data, since in
                that case Parcels deals with a better memory management during particle set execution.
                deferred_load=False is however sometimes necessary for plotting the fields.
+        :param field_chunksize: size of the chunks in dask loading
```

Comment:
Do we need to also copy this docstring to `FieldSet.from_netcdf` and similar methods in `fieldset.py`?